multimodal_agent/README.md

```diff
@@ -0,0 +1,110 @@
+# 🖼️ Multimodal Haystack Agent
+This demo shows how to build a multimodal agent that can interact with images and PDFs using [Haystack](https://haystack.deepset.ai/), [Hayhooks](https://github.com/deepset-ai/hayhooks), and [Open WebUI](https://docs.openwebui.com/).
+
+## 🚀 Run the Demo
+
+### 1️⃣ Clone the Repository
+```sh
+git clone git@github.com:deepset-ai/haystack-demos.git
+cd haystack-demos/multimodal_agent
+```
+### 2️⃣ Install Dependencies(Python>=3.10)
+Create and activate a virtual environment (optional but recommended):
+```sh
+python3.11 -m venv .venv
+source .venv/bin/activate
+```
+Install Haystack, Hayhooks and other required packages:
+```sh
+pip install haystack-ai python-weather hayhooks google-genai-haystack 
+```
+
+### 3️⃣ Configure Environment Variables
+Set API Keys:
+```sh
+export OPENAI_API_KEY=your_openai_api_key # For OpenAI models
+```
+
+🔗 **More details on configuration:** [Hayhooks Documentation](https://github.com/deepset-ai/hayhooks?tab=readme-ov-file#configuration)
+
+### 4️⃣ Start the Hayhooks Server 
+```sh
+hayhooks run
+```
+Check if Hayhooks is running:
+```sh
+hayhooks status
+```
+Output:
+```sh
+╭─────────────────────────────────────────────────────────────╮
+│ Hayhooks server is up and running at: http://localhost:1416 │
+╰─────────────────────────────────────────────────────────────╯
+
+No pipelines currently deployed
+```
+
+### 5️⃣ Deploy the Agent
+Choose the agent that you want to deploy and use the corresponding path:
+```sh
+hayhooks pipeline deploy-files -n basic-agent multimodal_agent/basic-agent
```

Comment:
This is not an Agent, but just a single-turn chatbot (does not handle chat history AFAIK)

Reply:
I removed this agent code